### PR TITLE
Update 201805.md

### PR DESCRIPTION
**Pull request by RichardRed0x**

### CODE REVIEW

File: posts/201805.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ In dcrd and dcrwallet developers have been [experimenting](https://github.com/de
 
 Decrediton: 1.2.1 bugfix [release](https://github.com/decred/decred-binaries/releases/tag/v1.2.1), work on SPV [has started](https://github.com/decred/decrediton/pull/1389), chart additions are in progress. Further simplification of the staking process is in the pipeline ([slack](https://decred.slack.com/archives/C5H9Z63AA/p1525528370000062)).
 
-Politeia: new [command line tool](https://github.com/decred/politeia/pull/267) to interact with Politeia API, general development is ongoing. Help with testing is very welcome: please check this [test plan](https://github.com/decred/politeia/issues/265) and join [#politeia](https://decred.slack.com/archives/C88HEA7DK) to get started.
+Politeia: new [command line tool](https://github.com/decred/politeia/pull/267) to interact with Politeia API, general development is ongoing. Help with testing will soon be welcome: this [test plan](https://github.com/decred/politeia/issues/265)  sets out a test plan, join [#politeia](https://decred.slack.com/archives/C88HEA7DK) to follow progress and partiicpate in testing.
```
Comment (xaur) [inline posts/201805.md]: participate

File: posts/201805.md [hunk 2]
```diff
@@ -26,13 +26,13 @@ Atomic swap with Ethereum on testnet was [demonstrated](https://twitter.com/bebr
...
 
-Staking: 30-day average ticket price is 91.3 DCR (+0.8), stake participation is 46.9% (+0.8%) with 3.68 million DCR locked (+0.15). Min price was 85.56. On May 11 ticket price surged to 96.99, staying elevated for longer than usual. Locked DCR peaked at 47.17%. This was [speculated](https://www.reddit.com/r/decred/comments/8ixay1/decred_investment_thesis_by_placeholder_ventures/dyvpgg5/?context=3) to be a result of a large miner staking.
+Staking: 30-day average ticket price is 91.3 DCR (+0.8), stake participation is 46.9% (+0.8%) with 3.68 million DCR locked (+0.15). Min price was 85.56. On May 11 ticket price surged to 96.99, staying elevated for longer than usual after such a pump. Locked DCR peaked at 47.17%. [nrpalao](https://www.reddit.com/user/nrpalao) on reddit [suggested](https://www.reddit.com/r/decred/comments/8ixay1/decred_investment_thesis_by_placeholder_ventures/dyvpgg5/?context=3) that the DCR for these tickets likely came from a miner with significant hashrate.
```
Comment (xaur) [inline posts/201805.md]: I like the idea of crediting reddit user, but nrpalao submitted Decred Investment Thesis, while isrly_eder noticed the whale staking and jet_user elaborated on possible (but very likely) connection with large miner.

File: posts/201805.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ Obelisk team [posted](https://medium.com/obelisk-blog/obelisk-update-january-may
 
 @SiaBillionaire [estimated](https://decred.slack.com/archives/C3P6R055Z/p1526221654000039) that 7940 DCR1 units were sold in Batches 1-5, while Lynmar13 [shared](https://medium.com/@brian7249694/the-current-state-of-the-obelisk-dcr1-fff97f6877be) his projections of DCR1 profitability ([reddit](https://www.reddit.com/r/decred/comments/8lgbv2/the_current_state_of_the_obelisk_dcr1/)).
 
-New Chinese miner was noticed by our Telegram group. Woodpecker WB2 specs 1.5 TH/s at 1200 W, costs 15,000 CNY (~2,340 USD) and the initial 150 units are expected to ship on Aug 15. ([pow8.com](https://pow8.com/product.php) – [translated](https://translate.google.com/translate?sl=auto&tl=en&hl=en&ie=UTF-8&u=https://pow8.com/product.php))
+A new Chinese miner for sale was noticed by our Telegram group. Woodpecker WB2 specs 1.5 TH/s at 1200 W, costs 15,000 CNY (~2,340 USD) and the initial 150 units are expected to ship on Aug 15. ([pow8.com](https://pow8.com/product.php) – [translated](https://translate.google.com/translate?sl=auto&tl=en&hl=en&ie=UTF-8&u=https://pow8.com/product.php))
```
Comment (xaur) [inline posts/201805.md]: Shall we use "pre-sale" to emphasize it will ship much later?

Comment (RichardRed0x) [reply posts/201805.md]: Sounds good

File: posts/201805.md (hunk 2 above)
Comment (RichardRed0x) [reply posts/201805.md]: Oops, got my names mixed up there.

**Comment by xaur** [comment]
Merged 307dd537

**Pull request closed without merging**